libconcurrent/includes/system.h

```diff
@@ -21,9 +21,6 @@
 #ifdef __GNUC__
 #    define CACHE_ALIGN __attribute__((aligned(CACHE_LINE_SIZE)))
 #    define VAR_ALIGN   __attribute__((aligned(16)))
-#elif defined(MSVC)
```

Comment:
What does this mean about Visual studio support?

Reply:
The very early versions of the code supported some intrinsics of MSVC. Since, supporting MSCV in the near future is not possible, these parts of the code are dead. Thus, we need to remove them.